packages/components/dashboard/resource-provider-card/resource-provider-card.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { HealthBody } from '@openshift-console/dynamic-plugin-sdk-internal';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Gallery,
+  GalleryItem,
+  Skeleton,
+} from '@patternfly/react-core';
+import {
+  NOOBAA_PROVIDER_MAP,
+  NS_PROVIDERS_NOOBAA_MAP,
+} from '../../../constants';
+import { NamespaceStoreKind } from '../../../types';
+import { nameSpaceStoreResource } from '../../resources';
+import './resource-provider-card.scss';
+
+export const ResourceProviderCard: React.FC = () => {
+  const { t } = useTranslation();
+  const [dataResources, dataResourcesLoaded, dataResourcesError] =
+    useK8sWatchResource<NamespaceStoreKind[]>(nameSpaceStoreResource);
+
+  const providerCount = React.useMemo((): { [key: string]: number } => {
+    const initialCount: { [key: string]: number } = Object.entries(
+      NS_PROVIDERS_NOOBAA_MAP
+    ).reduce((p, [k, v]) => Object.assign(p, { [k]: 0 }), {});
+    return dataResourcesLoaded && !dataResourcesError
+      ? dataResources?.reduce((drMap, dr) => {
+          const provider = dr?.spec?.type;
+          drMap[NOOBAA_PROVIDER_MAP[provider]] =
+            drMap[NOOBAA_PROVIDER_MAP[provider]] + 1;
+          return drMap;
+        }, initialCount)
+      : initialCount;
+  }, [dataResources, dataResourcesLoaded, dataResourcesError]);
```

Comment:
resource provider can be fetched from prometheus query. You will not require this logic